src/providers/app.tsx

```diff
@@ -38,7 +39,9 @@ function AppProvider({ children }: { children: ReactNode }) {
           >
             <Notifications />
             <InitProvider>
-              <AuthProvider>{children}</AuthProvider>
+              <ModalsProvider labels={{ confirm: 'Submit', cancel: 'Cancel' }}>
```

Comment:
I think remove the labels part. I doubt it is 'Submit' very often. I can see the confirm label being 'Create' or 'Confirm' or lots of other options. I think it is better to be explicit so just remove labels={{ confirm: 'Submit', cancel: 'Cancel' }}